dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -31,8 +30,9 @@ import NavItems from "../NavbarComponent";
 import EmptyTable from "../EmptyStateComponent";
 import moment from "moment";
 import { fetchPublicDatasets } from "../../../actions/fetchPublicDatasets";
+import TablePagination from "../PaginationComponent";
 let startDate = moment(new Date(1990, 10, 4)).format("YYYY/MM/DD");
-let endDate = moment(new Date(2040, 10, 4)).format("YYYY/MM/DD");
+let endDate = moment(new Date()).format("YYYY/MM/DD");
```

Comment:
Do we want the "YYYY/MM/DD" format standard in the UI, or should we go for "YYYY-MM-DD"? The best plan for now might be to centralize this conversion as a utility function so that we can change it in one place.